src/platform/plugins/shared/controls/public/controls/data_controls/options_list_control/components/options_list_popover_suggestions.tsx

```diff
@@ -197,7 +198,7 @@ export const OptionsListPopoverSuggestions = ({
             selectableOptions.length
           )}
           emptyMessage={<OptionsListPopoverEmptyMessage showOnlySelected={showOnlySelected} />}
-          onChange={(newSuggestions, _, changedOption) => {
+          onChange={(_, __, changedOption) => {
```

Comment:
even though these are used, how about providing names so we know what they are?

Reply:
Done in c76866c429ee4f721561870360cadd99ee9d88fc :+1: